src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --colors-link: rgba(0, 172, 220, 1);
+}
+
 body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+header {
+  width: 100vw;
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+nav {
+  display: flex;
+  position: relative;
+  margin: 0;
+  right: 50px;
```

Comment:
Using `right: 50px;` for positioning the `nav` element might not be responsive. Consider using a more flexible layout approach, such as flexbox or grid, to ensure the navigation adapts well to different screen sizes.